kernel/shared/src/main/scala/monocle/Optional.scala

```diff
@@ -28,4 +29,7 @@ object Optional {
 
   def tailOption[S](implicit ev: Cons[S]): Optional[S, S] =
     ev.tailOption
+
+  def possible[S, A](implicit ev: Possible.Aux[S, A]): Optional[S, A] =
```

Comment:
Is this really useful?
A related question why is `headOption[S, A]: Optional[S, A]` while `tailOption[S]: Optional[S, S]`?

Reply:
I think it is. It is a way to summon a `possible` instance such as we can do:
```scala
foo compose Optional.possible
```

I believe we can offer a much better syntax in another module. I will send a proof of concept PR soon

Reply:
Do you think I should remove it?

Reply:
ah no, I think it is useful